src/app/users/users-profile/users-profile.component.ts

```diff
@@ -68,7 +70,9 @@ export class UsersProfileComponent implements OnInit {
       return of({ ok: false, reason: 'Error changing password' });
     })).subscribe((res) => {
       if (res.ok === true) {
-        // TODO: Should notify user that password successfully changed or that there was an error
+        this.planetMessageService.showAlert('Password sucessfully changed');
+      } else {
+        this.planetMessageService.showAlert('There was an error');
```

Comment:
this could show res.reason on message as it already has message return from `return of({ ok: false, reason: 'Error changing password' });`

Reply:
OK